lib/core/text/flat.nit

```diff
@@ -1308,37 +1308,44 @@ private class FlatBufferCharIterator
 end
 
 redef class CString
-	redef fun to_s
-	do
-		return to_s_with_length(cstring_length)
-	end
 
-	redef fun to_s_with_length(length)
-	do
-		assert length >= 0
-		return clean_utf8(length)
-	end
+	# Get a `String` from the data at `self` copied into Nit memory
+	#
+	# Require: `self` is a null-terminated string.
+	redef fun to_s do return to_s_unsafe
```

Comment:
I'm not sure the `unsafe` version should be the default as it might introduce errors when used by developers unaware of Unicode issues, maybe the `safe` version would be better?